src/components/utility/SocialShareToolbar.scss

```diff
@@ -26,6 +37,13 @@
       &__item {
         padding: 5px;
       }
+
+      .SocialMediaShareButton{

```

Comment:
Please watch the missing space. I guess we should start introducing css linting.